target/linux/mediatek/dts/mt7986a-asus-rt-ax59u.dts

```diff
@@ -22,7 +23,6 @@
 
 	chosen {
 		stdout-path = "serial0:115200n8";
-		bootargs-override = "ubi.mtd=UBI_DEV";
```

Comment:
This seems unrelated to described changes

Reply:
It is related because due to the added `compatible = "linux,ubi";` below which replaces the cmdline parameter.